### DISCUSSION TITLE
Change da world

### DISCUSSION THREAD

**Discussion by mashed-potatoes** [kitsuned/PotatoNV#73] [General]
- New UI, no more textboxes
- Fixed crash due to an invalid response in VCOM mode
- Fixed `WVLOCK` readback
- Fixed `FBLOCK` state detection on legacy devices (#63 – thx @TBM13)

<hr /><em>This discussion was created from the release <a href='https://github.com/mashed-potatoes/PotatoNV/releases/tag/v2.2.1'>Change da world</a>.</em>

**Comment by Devlin9228** [comment]
А 710 будет?)

**Comment by panuganties** [comment]
please add kirin 655 (honor 6x)
